dvclive/live.py

```diff
@@ -15,7 +15,7 @@
     InvalidDataTypeError,
     InvalidPlotTypeError,
 )
-from .report import html_report
+from .report import make_report
 from .utils import env2bool, nested_update, open_file_in_browser
```

Comment:
Is it OK to deprecate an attribute just like that? Should it stay hidden for backward compatibility instead?

Reply:
@daavoo WDYT? Since we are still pre-1.0 (we probably need to consider a 1.0 release), we can technically break anything. Not sure how you feel about keeping an alias for backward compatibility.

Reply:
In this case, i feel its not necesary. The public and documented method is/was `Live.make_report` which didn't change. This was an internal función.

Regarding 1.0 I added items to https://github.com/iterative/dvclive/issues/223